adapters/sharethrough/butler.go

```diff
@@ -71,7 +71,6 @@ func (s StrOpenRTBTranslator) requestFromOpenRTB(imp openrtb.Imp, request *openr
 	headers := http.Header{}
 	headers.Add("Content-Type", "application/json;charset=utf-8")
 	headers.Add("Accept", "application/json")
-	headers.Add("Accept-Encoding", "gzip")
```

Comment:
Curious, why is gzip no longer accepted?

Reply:
@SyntaxNode it actually broke our adapter, our ad server now returns gzip encoded responses and it looks like the adapter needs to handle unzipping itself. We'll push a fix later to decode gzipped responses but for now we just need this fix asap in production as it can't process any bid in its current state.

Reply:
Gotcha.